### ISSUE TITLE
Setup Docker

### ISSUE THREAD

**Issue by DesignrKnight** [kautukkundan/omg-badges#3]
Solve the problem of "it works on my machine" and have a unified setup to work in all platforms

**Comment by DesignrKnight** [comment]
I would like to work on this

**Comment by kautukkundan** [comment]
Sure! Go ahead.

**Comment by kautukkundan** [comment]
@DesignrKnight are you still on it?

**Comment by DesignrKnight** [comment]
Yes, I did set up docker and deployed also, I will send PR this weekend. Full one click to deploy ready
